content/docs/pipelines/pipelines-tutorial.md

```diff
@@ -0,0 +1,616 @@
+
+Follow these steps to set up your GCP environment:
+
+1. Select or create a project on the [GCP Console][gcp-console].
+1. Make sure that billing is enabled for your project. See the guide to
+  [modifying a project's billing settings][billing-guide].
+1. Install the [Cloud SDK][cloud-sdk].
+
+Notes:
+
+* As you work through this tutorial, your project uses billable components of
+  GCP. To minimise costs, follow the instructions to
+  [clean up your GCP resources](#cleanup) when you've finished with them.
+* This guide assumes you want to manage your GCP environment on your own server
+  rather than in the [Cloud Shell][cloud-shell] environment. If you choose to
+  use the Cloud Shell, some of the components are pre-installed in your shell.
+
+### (Optional) Install kubectl
+
+`kubectl` is the command-line tool for Kubernetes. `kubectl` is useful if you 
+want to interact with your Kubeflow cluster locally. If you decide not to use
+`kubectl`, you can skip the steps later in the tutorial that use the command.
+
+Run the following Cloud SDK command to install `kubectl`:
+
+```
+gcloud components install kubectl
+```
+
+### Set up some handy environment variables
+
+Set up the following environment variables for use throughout the tutorial:
+
+1. Set your GCP project ID. In the command below, replace `<YOUR-PROJECT-ID>` 
+  with your [project ID][gcp-project-id]:
+
+    ```
+    export PROJECT=<YOUR-PROJECT-ID>
+    gcloud config set project ${PROJECT}
+    ```
+
+1. Set the zone for your GCP configuration. Choose a zone that offers the
+  resources you need. See the guide to [GCP regions and zones][regions-zones].
+    * Ensure you have enough Compute Engine regional capacity.
+      By default, the GKE cluster setup described in this guide
+      requires 16 CPUs.
+    * If you want a GPU, ensure your zone offers GPUs.
+
+    For example, the following commands set the zone to `us-central1-c`:
+
+    ```
+    export ZONE=us-central1-c
+    gcloud config set compute/zone ${ZONE}
+    ```
+
+1. If you want a custom name for your Kubeflow deployment, set the 
+   `DEPLOYMENT_NAME` environment variable. The deployment name must be 
+   **4-20 characters** in length. Note that the name must be the same 
+   as the one you use in later steps of this tutorial when configuring the 
+   **redirect URI** for the OAuth client credentials. If you don't set this 
+   environment variable, your deployment gets the default name of `kubeflow`:
+
+    ```
+    export DEPLOYMENT_NAME=kubeflow
+    ```
+
+### Deploy Kubeflow
+
+Deploy Kubeflow on GCP:
+
+1. Follow the instructions in the 
+  [GKE getting-started guide](/docs/started/getting-started-gke), 
+  taking note of the following:
+
+  * Set up **OAuth client credentials** and **Cloud Identity-Aware Proxy (IAP)**
+    as prompted during the deployment process. So, **do not choose the deployment 
+    option to skip IAP**. IAP ensures you can connect securely to the Kubeflow
+    web applications.
```

Comment:
I think this bullet can be compressed to just "Ensure you have the IAP enabled" or similar. The getting started guide should explain IAP in more detail, and that would make this bullet easier to digest

Reply:
Done.